teachopencadd/talktorials/T013_query_pubchem/talktorial.ipynb

```diff
@@ -9,7 +9,8 @@
     "Authors:\n",
```

Comment:
<p>Do we need to return the input <code>cids</code>  again? I think it would be enough with just the obtained SMILES.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/volkamerlab/teachopencadd/pull/81/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='26'/>

Reply:
<p><code>cid</code>  has been removed.</p>